---
layout: default
title: Biomedical
nav_order: 2
parent: Data Sources
---

{:.no_toc}
# Biomedical

* TOC
{:toc}

### [ELIXIR Core Data Resources](https://elixir-europe.org/platforms/data/core-data-resources)

#### [The Molecular INTeraction (MINT) Database](https://mint.bio.uniroma2.it/)
The MINT Database "focuses on experimentally verified protein-protein interactions mined from the scientific literature by expert curators."

Licata, Luana, Leonardo Briganti, Daniele Peluso, Livia Perfetto, Marta Iannuccelli, Eugenia Galeota, Francesca Sacco et al. "MINT, the molecular interaction database: 2012 update." Nucleic acids research 40, no. D1 (2012): D857-D861.


### [Encyclopedia of DNA Elements (ENCODE)](https://www.encodeproject.org/)

#### [BED (Browser Extensible Data) Files](https://www.encodeproject.org/help/project-overview/)
The ENCODE dataset contains information for approximately 7000 experiments along with 14,000 BED files collected by The Encyclopedia of DNA Elements (ENCODE) Consortium. Examples of experiment metadata captured include the target biosample, assay type, gene assembly, etc. Bed files link to individual bed lines, which state the genomic position of individual peaks. Data Commons ingested all experimental data in BED format.

Data made available under: [ENCODE Data Use Policy for External Users](https://www.encodeproject.org/help/citing-encode/). This data was formatted for Data Commons through a collaboration with Dr. Anthony Oro’s group at Stanford University.


### [European Molecular Biology Laboratory - European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/)

#### [ChEMBL](https://www.ebi.ac.uk/chembl/)
"ChEMBL is a manually curated database of bioactive molecules with drug-like properties. It brings together chemical, bioactivity and genomic data to aid the translation of genomic information into effective new drugs." It includes information on drugs at all stages of drug discovery.

This data is made available by [EMBL-EPI Terms of Use](https://www.ebi.ac.uk/about/terms-of-use/). This data was formatted for Data Commons in part through a collaboration with Dr. Sergio Baranzini’s group at UCSF.


### [Genotype-Tissue Expression (GTEx)](https://www.gtexportal.org/home/)

#### [GTEx Analysis V8](https://www.gtexportal.org/home/datasets)
The GTEx eGene and significant variant-gene association data were generated from samples "collected from 54 non-diseased tissue sites across nearly 1000 individuals, primarily for molecular assays including WGS, WES, and RNA-Seq. Remaining samples are available from the GTEx Biobank." The single-tissue cis-eQTL data from the v8 release was used. Due to the size of the datasets only Skin - Not Sun Exposed and Skin - Sun Exposed are made available on the main graph. The data for all tissues can be accessed on the Biomedical Data Commons knowledge graph.

GTEx is an NIH human genomic data unrestricted-access data repository and the data was made available in compliance with [GTEx Data Release and Publication Policy](https://www.gtexportal.org/home/documentationPage#staticTextPublicationPolicy). GTEx outlines [how to cite](https://www.gtexportal.org/home/faq#citePortal) use of GTEx data in journal publication.


### [Institute of Genome Sciences](https://www.igs.umaryland.edu/)

#### [The Disease Ontology](https://disease-ontology.org/)
The Disease Ontology was developed as a project by the Institute of Genome Sciences at the University of Maryland School of Medicine. It "is a community driven, open source ontology that is designed to link disparate datasets through disease concepts". It provides a "standardized ontology for human disease with the purpose of providing the biomedical community with consistent, reusable and sustainable descriptions of human disease terms, phenotype characteristics and related medical vocabulary disease concepts".

The data is made available under [C0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://disease-ontology.org/resources/citing-do). Data Commons includes the 3/7/19 update of the Disease Ontology. This data was formatted for Data Commons through a collaboration with Dr. Sergio Baranzini’s group at UCSF.


### [New York Botanical Garden (NYBG)](http://sweetgum.nybg.org/science/)

#### [C. V. Starr Virtual Herbarium (Collaboration)](http://sweetgum.nybg.org/science/vh/learn-more/)
C. V. Starr Virtual Herbarium is a public specimen database with photos and detailed records about millions of plants, fungi, and algae.


### [Proteomics Standards Initiative](https://psidev.info/)

#### [HUPO-PSI Working Groups and Outputs](https://psidev.info/)
The Molecular Interactions Controlled Vocabulary from the HUPO Proteomics Standards Initiative working groups is "a structured controlled vocabulary for the annotation of experiments concerned with protein-protein interactions". The ontologies dictionary is represented in a tree structure in the [EMBL-EBI Ontology Lookup Service](https://www.ebi.ac.uk/ols/ontologies/mi). Data Commons includes three subsets of the ontologies: "interaction detection method", "interaction type" and "database citation", which are commonly used in protein-protein interactions.

Data Made available under [Apache License 2.0](https://github.com/EBISPOT/OLS/blob/main/LICENSE). The license information of HUPO PSI can be found at the [Community Practice](http://www.psidev.info/sites/default/files/CommunityPractice-revised.doc). See also [EBI term of use](http://www.ebi.ac.uk/about/terms-of-use/).


### [Side Effect Resource (SIDER)](http://sideeffects.embl.de/)

#### [SIDER 4.1](http://sideeffects.embl.de/)
SIDER is a database of adverse drug reactions curated by the EMBL collaboration. "SIDER contains information on marketed medicines and their recorded adverse drug reactions. The information is extracted from public documents and package inserts. The available information include side effect frequency, drug and side effect classifications as well as links to further information, for example drug–target relations." Data Commons hosts version 4.1 of SIDER released on October 21, 2015.

This data is made available under the [Creative Commons Attribution-Noncommercial-Share Alike 4.0 License](https://creativecommons.org/licenses/by-nc-sa/4.0/). Information about citing SIDER can be found [here](http://sideeffects.embl.de/about/). This data was formatted for Data Commons through a collaboration with Dr. Sergio Baranzini’s group at UCSF.


### [The Human Protein Atlas](https://www.proteinatlas.org/)

#### [The Tissue Atlas](https://www.proteinatlas.org/humanproteome/tissue)
The Human Protein Tissue Atlas contains information about the distribution of proteins on human tissues derived from the antibody-based protein profiling from 44 normal human tissues types and mRNA expression data from 37 different normal tissue types.

This [dataset](https://www.proteinatlas.org/download/normal_tissue.tsv.zip) is available under [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/). Please also see their [Disclaimer](https://www.proteinatlas.org/about/disclaimer) and [Licence & Citation](https://www.proteinatlas.org/about/licence).


### [U.S. Food and Drug Administration (FDA)](https://www.fda.gov/)

#### [FDA-Approved Drugs](https://www.accessdata.fda.gov/scripts/cder/daf/)
"Drugs@FDA includes information about drugs, including biological products, approved for human use in the United States." Data Commons includes the information about the FDA application for the drug as well as the drug’s strength, active ingredients, dosage forms, administration routes, FDA therapeutic equivalence code, and marketing status.


#### [Pharmacologic Class](https://www.fda.gov/industry/structured-product-labeling-resources/pharmacologic-class)
The FDA established pharmacologic classes "associated with an approved indication of an active moiety that the FDA has determined to be scientifically valid and clinically meaningful". This includes the (1) description of pharmacologic class (2) active moiety code and description (3) compounds associated with each class.

This data is made available through [openFDA terms of service](https://open.fda.gov/license/).


### [U.S. National Institutes of Health: National Center for Biotechnology Information](https://www.ncbi.nlm.nih.gov/)

#### [ClinVar](https://www.ncbi.nlm.nih.gov/clinvar/)
"ClinVar is a freely accessible, public archive of reports of the relationships among human variations and phenotypes, with supporting evidence." It contains reports of genetic "variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data." Data Commons includes the January 6, 2020 release of the ClinVar archive supporting both hg19 and hg38 genome assemblies.


#### [Gene](https://www.ncbi.nlm.nih.gov/gene)
The NIH NCBI gene info datasets from NCBI Gene for a subset of species contains "gene-specific content based on NCBI's RefSeq project, information from model organism databases, and links to other resources." The NCBI RefSeq project is "a comprehensive, integrated, non-redundant, well-annotated set of reference sequences including genomic, transcript, and protein". The datasets included are from the February 19, 2020 update. The gene info files for the following species have been added:

* _Caenorhabditis elegans_
* _Danio rerio_
* _Drosophila melanogaster_
* _Gallus gallus_
* _Homo sapiens_
* _Mus musculus_
* _Saccharomyces cerevisiae_
* _Xenepus laevis_

This data is from an NIH human genome unrestricted-access data repository and made accessible under the [NIH Genomic Data Sharing (GDS) Policy](https://osp.od.nih.gov/scientific-sharing/genomic-data-sharing/).


### [U.S. National Institutes of Health: National Library of Medicine](https://www.nlm.nih.gov/)

#### [Medical Subject Headings (MeSH)](https://meshb.nlm.nih.gov/search)
"The Medical Subject Headings (MeSH) thesaurus is a controlled and hierarchically-organized vocabulary produced by the National Library of Medicine. It is used for indexing, cataloging, and searching of biomedical and health-related information". Data Commons includes the Descriptor, Concept, and Term elements of MeSH as described [here](https://www.nlm.nih.gov/mesh/xml_data_elements.html).

This data is from the National Library of Medicine (NLM) and is not subject to copyright and is freely reproducible as stated in the [NLM’s copyright policy](https://www.nlm.nih.gov/web_policies.html). This data was formatted for Data Commons through a collaboration with Dr. Sergio Baranzini’s group at UCSF.


### [UCSC Genomics Institute](https://genome.ucsc.edu/)

#### [Genome Browser](https://genome.ucsc.edu/)
The UCSC Genome Browser originated from The Human Genome Project in 2000 to share and visualize genome data. It has grown to include an agglomeration of various genome assemblies and annotations. Data Commons includes data annotating chromosomes, genes, RNA transcripts, and genetic variants from the UCSC Genome Browser. The .chrom.sizes.txt files were downloaded from the UCSC Genome Browser Downloads page on August 13, 2019. The NCBI RefSeq files were downloaded from the UCSC Table Browser on August 2, 2019 for the following genome assemblies:

* ce10
* ce11
* danRer10
* danRer11
* dm3
* dm6
* galGal5
* galGal6
* hg19
* hg38
* mm9
* mm10
* sacCer3
* xenLae2

The All SNPs files were downloaded from the UCSC Table Browser on August 13, 2019 for the following genome assemblies and dbSNP builds:

* gaGal5 (dbSNP Build 147)
* hg19 (dbSNP Build 151)
* hg38 (dbSNP Build 151)
* mm9 (dbSNP Build 128)
* mm10 (dbSNP Build 142)

The annotation data is made freely available under the UCSC Genome Browser [terms of use](https://genome.ucsc.edu/conditions.html). The UCSC Genome Browser states [how to cite](https://genome.ucsc.edu/cite.html) use of their data in a journal article publication.


### [UniProt](https://www.uniprot.org/)
Data Commons includes protein sequence and functional information including protein interaction with chemical compounds maintained by the UniProt Consortium.


#### [UniProt Controlled Vocabulary of Species](https://www.uniprot.org/docs/speclist)
UniProt’s Controlled Vocabulary of Species contains organism species UniProt identification codes, [NCBI Taxonomy database identifiers](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi), scientific names, common names, synonyms, and organism kingdoms.

The data is made available by the [Creative Commons Attribution (CC BY 4.0) License](https://creativecommons.org/licenses/by/4.0/). Further information on UniProt License and Disclaimer can be found [here](https://www.uniprot.org/help/license). The UniProt Consortium states [how to cite](https://www.uniprot.org/help/publications) Uniprot data used in a journal article. This data was formatted for Data Commons in part through a collaboration with Dr. Sergio Baranzini’s group at UCSF.


### [University of California San Francisco](https://www.ucsf.edu/)

#### [SPOKE Disease Symptom Associations](https://spoke.ucsf.edu/)
These are statistical associations using a Fisher’s exact test co-occurrence of disease and symptom terms in Pubmed entries by performing as described in [Himmelstein, et al (2017)](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640425/).

The data was previously hosted by [UCSF Scalable Precision Medicine Knowledge Engine SPOKE](https://spoke.ucsf.edu/). It was made available by the data’s owner, Sergio Baranzini, for use on Data Commons. This data was formatted for Data Commons through a collaboration with Dr. Sergio Baranzini’s group at UCSF.
